native/cocos/scene/RenderWindow.h

```diff
@@ -65,8 +65,8 @@ class RenderWindow : public RefCounted {
 
     void extractRenderCameras(ccstd::vector<Camera *> &cameras);
 
-    void onNativeWindowDestroy(uint32_t windowId);
-    void onNativeWindowResume(uint32_t windowId);
+    void onNativeWindowDestroy(void *windowId);
+    void onNativeWindowResume(void *windowId);
```

Comment:
Why use void*?

Reply:
I use ```void*``` to match args from ```EventDispatcher```. It seems a little weird DESTROY_EVENT use windowHandle while RESUME_EVENT use windowId.

Reply:
@freeboy i think should use window id both in resume and destroy event.